src/box/lua/console.lua

```diff
@@ -278,6 +278,12 @@ local function set_language(storage, value)
         local msg = 'Invalid language "%s", supported languages: lua and sql.'
         return error(msg:format(value))
     end
+    if value == 'sql' then
+        if rawget(box, 'execute') == nil then
+            local msg = "Unable to set language to 'sql' with unconfigured box"
+            return error(msg:format(value))
```

Comment:
Nit: `format()` is pointless here, please remove. Also, I'd put the whole condition in one `if`:

```Lua
if value == 'sql' and rawget(box, 'execute') == nil then
```

Reply:
True, sorry. Fixed.